src/js/components/MainMenu.vue

```diff
@@ -133,31 +140,34 @@ export default {
       menuBarsTooltip: "",
       queryString: '',
       menuDisplayInstitutional: true,
-      menu: 'col',
+      menuDisplay: 'text',
     }
   },
   methods: {
     onArtifactSelected(args) {
+    },
+    loadCombinations() {
+      if (this.$store.getters.sessionID && this.$store.getters.userID > -1) {
+        this.combinations = []
+        this.$post('resources/cgi-bin/GetImageData.pl', {
+          transaction: 'getCombs',
+          user: this.$store.getters.userID,
+          SESSION_ID: this.$store.getters.sessionID
+        })
+        .then(res => {
+          if (res.status === 200 && res.data) {
+            this.combinations = res.data.results
+          }
+        })
+        .catch(console.log)
```

Comment:
For the moment, I'm happy with `console.log` or (probably preferable) `console.error` for these `.catch` blocks, but eventually we'll put in place some more rigorous error handling.